api/src/main/java/ai/djl/ndarray/NDArrays.java

```diff
@@ -1989,4 +1989,110 @@ public static NDArray logicalXor(NDArray a, NDArray b) {
     public static NDArray erfinv(NDArray input) {
         return input.erfinv();
     }
+
```

Comment:
We don't need to add norm to NDArrays. NDArrays only has methods that involve multiple NDArrays where it may be odd to associate the method with just the first one. Since norm only uses a single NDArray, we can just keep it on NDArray.

Reply:
I see. I'll remove it.